src/pages/index.js

```diff
@@ -119,17 +214,36 @@ function handleEscapePress(evt) {
   }
 }
 
-
 function handleAddCardSubmit(evt) {
   evt.preventDefault(settings);
   console.log(editModalNameInput.value);
   console.log(editModalDescriptionInput.value);
-  const inputValues = { name: cardCaptionInput.value, link: cardLinkInput.value };
-  const cardEl = getCardElement(inputValues);
-  cardsList.prepend(cardEl);
-  evt.target.reset();
-  disableButton(cardSubmitBtn, settings);
-  closeModal(cardModal, settings);
+  const inputValues = {
+    name: cardCaptionInput.value,
+    link: cardLinkInput.value,
+  };
+
+  api.addCard(inputValues).then(() => {
+    const cardEl = getCardElement(inputValues);
+    cardsList.prepend(cardEl);
+    evt.target.reset();
+    disableButton(cardSubmitBtn, settings);
+    closeModal(cardModal, settings);
+  });
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere in the code**